docs/src/main/sphinx/release/release-383.md

```diff
@@ -0,0 +1,128 @@
+# Release 383 (1 June 2022)
+
+## General
+
+* Introduce `json_exists`, `json_query`, and `json_value` functions. ({issue}`9081`)
+* Add AWS IAM Role support for exchange spooling on S3. ({issue}`12444`)
+* Improve query performance by reducing worker-to-worker communication overhead. ({issue}`11289`)
+* Improve performance and reduce memory usage of queries that contain aggregations. ({issue}`12336`)
+* Improve performance of correlated queries involving distinct aggregations. ({issue}`12564`)
+
+## Security
+
+## Web UI
+* Improve readability of cumulative user memory on query details page.
+
+## JDBC driver
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+## Accumulo connector
+
+* Fail creating a new table if a table comment or a column comment is specified.
+  Previously, the comment was ignored. ({issue}`12452`)
+
+## Atop connector
+
+## BigQuery connector
+
+* Fail creating a new table if a table comment or a column comment is specified.
+  Previously, the comment was ignored. ({issue}`12452`)
+
+## Blackhole connector
+
+## Cassandra connector
+
+* Fail creating a new table if a table comment or a column comment is specified.
+  Previously, the comment was ignored. ({issue}`12452`)
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+* Support storing a table comment when creating new tables. ({issue}`12452`)
```

Comment:
creating a new table.

Also .. does that include column comments?

Reply:
I don't think it includes column comments yet. This is for #12452, which was only table comments.